### PR TITLE
:checkered_flag: Updated content post sign-off

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: test/integration/choosePage.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ describe('Choose page', () => {
     it('page options should be related to being over 25 (paid), if age question is answered 25 or older', async () => {
       const $ = cheerio.load(res.text);
 
-      expect($($('.multiple-choice .multiple--choice-option')[0]).text()).to.equal('See a sexual health professional for free near you');
+      expect($($('.multiple-choice .multiple--choice-option')[0]).text()).to.equal('See a sexual health professional near you');
```
Comment (markysoft) [inline test/integration/choosePage.js]: You can simplify this by using first, i.e.
`expect($('.multiple-choice .multiple--choice-option').first().text()).to.equal('See a sexual health professional near you');`

File: test/integration/choosePageErrors.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ describe('Choose page with error', () => {
     it('page options should be related to being over 25 (paid), if errors and age is over 25', async () => {
       const $ = cheerio.load(res.text);
 
-      expect($($('.multiple-choice .multiple--choice-option')[0]).text()).to.equal('See a sexual health professional for free near you');
+      expect($($('.multiple-choice .multiple--choice-option')[0]).text()).to.equal('See a sexual health professional near you');
```
Comment (markysoft) [inline test/integration/choosePageErrors.js]: See above comment

File: test/integration/recommendPage.js [hunk 3]
```diff
@@ -76,7 +76,7 @@ describe('Recommend page', () => {
     iExpect.htmlWith200Status(res);
     const $ = cheerio.load(res.text);
 
-    expect($($('.page-section p')[3]).text()).to.equal('This is because you’re under the legal age of consent for sex.');
+    expect($($('.page-section p')[3]).text()).to.equal('We\'re not able to offer test kits if you\'re under 16. You\'ll need to see a health professional to get tested.');
```
Comment (markysoft) [inline test/integration/recommendPage.js]: Rather than use two $ wrappers, you could use:
`$('.page-section p').eq(3).text()`

**Comment by st3v3nhunt** [comment]
@markysoft good spots. I've updated (even though I didn't change it 😉 ).

**Comment by st3v3nhunt** [comment]
@markysoft There is another tweak to make to the content. I'll get it sorted before this is merged, if thats ok?

**Comment by st3v3nhunt** [comment]
Change made, it was to remove a period.

**Pull request merged**